.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,47 @@
+<!--
+This form is for bug reports and feature requests ONLY!
+
+For high severity issues, please contact AWS Support directly.
+
+Please provide the following information:
+-->
+
+<!--
+BUG REPORT or FEATURE REQUEST ?
+
+Uncomment only one of the below items
+-->
+
+<!-- ### Bug Report -->
+<!-- ### Feature Request -->
+
+
+### Summary
+<!-- Please provide a brief outline of the bug or feature request -->
+
+
+### Description
+<!-- Provide detailed information about the issue -->
+
+
+<!-- -->
+### Expected Behavior
+
+
+### Supporting Log Snippets
+<!-- Please provide supporting log information for this issue -->
+
+
+### Stacktrace
+
+
+### Environment Details
+
+
+### Additional Information
+<!--
+Please have a look at https://github.com/awslabs/ecs-logs-collector
```

Comment:
No, this isn't the same as the log section above. 
It is always useful to have a section dedicated to miscellaneous information. 

In the case of feature requests, this could be used to point to relevant references, documentation, research papers, etc...
In the case of bugs, this section could help document information not captured above. 

Perhaps, we should rename it to miscellaneous information ?